public/src/node.js

```diff
@@ -0,0 +1,146 @@
+// # Analyzing Data with Node
+//
+// As mentioned in the [introduction](getting_started.html), this guide is mostly geared for client-side data analysis, but with a few augmentations, the same tools can be readily used server-side with [Node](https://nodejs.org/).
+//
+// And the data is too large, this might in fact be your only option if you want to use JavaScript for your data analysis. None of your users will wait 5 minutes on your site while a giant file is loaded up, no matter how cool the analysis.
+//
+// ## Setting up a Node Project
+//
+// To get started with Node, ensure both node and [npm](https://www.npmjs.com/), the Node package manager, are installed and available via the command line:
+//
+// ```bash
+// which node
+// # /usr/local/bin/node
+// which npm
+// # /usr/local/bin/npm
+// ```
+//
+// Your paths may be different then mine, but as long as `which` returns something, you should be good to go. 
+//
+// Create a new directory for your data analysis project. In this example, we have a directory with a sub-directory called `data` which contains our `animals.tsv` file inside.
+//
+// ## Installing Node Modules
+//
+// Next, we want to install our JavaScript tools, D3 and lodash. With Node, we can automate the process by using `npm`. Inside your data analysis directory run the following:
+//
+// ```bash
+// npm install d3
+// npm install lodash
+// ```
+//
+// You can see that npm creates a new sub-directory called `node_modules` by default, where your packages are installed. Everything is kept local, so you don't have to worry about problems with missing or out-of-date packages. Your analysis tools for each project are ready to go.
```

Comment:
sure. maybe with a link on if they want to know more... i'll see if i can find something good.